ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/README.metadata.json

```diff
@@ -23,7 +23,9 @@
     "snippets": [
         "DisplayScenesInTabletopAR.cpp",
         "DisplayScenesInTabletopAR.h",
-        "DisplayScenesInTabletopAR.qml"
+        "DisplayScenesInTabletopAR.qml",
+        "PermissionsHelper.cpp",
+        "PermissionsHelper.h"
     ],
     "title": "Display scenes in tabletop AR"
 }
```

Comment:
Does it matter if this file doesn't have a trailing newline? GitHub thinks it's a problem.

Reply:
It hasn't been a problem in the past so I would guess not. Just looks like GitHub providing additional information.